install-casaos/run.sh

```
@@ -551,6 +552,169 @@ Check_Docker_Snap() {
+
+# Check and fix containerd version for LXC/Proxmox
+Check_Containerd_LXC_Fix() {
+    Show 2 "Checking for LXC/Proxmox environment..."
+    
+    if Check_LXC_Environment; then
+        Show 3 "LXC/Proxmox environment detected"
+        echo -e "${aCOLOUR[4]}Checking containerd.io version for CVE-2025-52881 compatibility...${COLOUR_RESET}"
+        
+        # Get current containerd version
+        if command -v containerd &>/dev/null; then
+            local current_version=$(dpkg -l | grep containerd.io | awk '{print $3}' | head -n1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)